modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/views/table/Table.js

```diff
@@ -210,9 +210,7 @@ function Table({dataLoading, items, itemsActions, schema, style}) {
 														item.actions) && (
 														<Actions
 															actions={
-																itemsActions ||
-																item.actions ||
-																item.actionDropdownItems
+																itemsActions
```

Comment:
item.actions, item.actionDropdownItems and itemsActions are the same?

Reply:
No, but this condition makes no sense. `itemsActions` will never be falsy, as the default is an empty array:
```js
[] || 'foo' || 'bar' // []
```
We are not supporting action items definition in items themselves anyways. This is likely leftover code from some previous solution.